src/smo/data/music.ts

```diff
@@ -1666,6 +1666,24 @@ export class SmoMusic {
     stemTicks = stemTicks * 2;
     return SmoMusic.ticksToDuration[stemTicks];
   }
+  //todo: figure out a better name
+  // ## closestSmoDuration
```

Comment:
How about closestBeamDuration.  Note duration can be indicated by beam or notehead or flag or stem, but beam seems like a good label, whether the actual note is beamed or not.  I use vex length elsewhere to mean a duration that can be rendered as a beam duration + dots (that could also have a better name)